src/Protocol/Diaspora.php

```diff
@@ -1476,23 +1476,29 @@ private static function message($guid, $server, $level = 0)
 	public static function fetchByURL($url, $uid = 0)
 	{
 		// Check for Diaspora (and Friendica) typical paths
-		if (!preg_match("=(https?://.+)/(?:posts|display)/([a-zA-Z0-9-_@.:%]+[a-zA-Z0-9])=i", $url, $matches)) {
+		if (!preg_match("=(https?://.+)/(?:posts|display|objects)/([a-zA-Z0-9-_@.:%]+[a-zA-Z0-9])=i", $url, $matches)) {
+			Logger::info('Invalid url', ['url' => $url]);
 			return false;
 		}
 
 		$guid = urldecode($matches[2]);
 
 		$item = Item::selectFirst(['id'], ['guid' => $guid, 'uid' => $uid]);
 		if (DBA::isResult($item)) {
+			Logger::info('Found', ['id' => $item['id']]);
```

Comment:
Shouldn't you add the `$url` in the logger context?

Reply:
The log data had more the purpose if the item had been found - or not. But yes, they could be improved.